tests/queries/0_stateless/00121_drop_column_zookeeper.sql

```diff
@@ -1,14 +1,14 @@
 -- Tags: zookeeper, no-replicated-database
 -- Tag no-replicated-database: Old syntax is not allowed
 
-DROP TABLE IF EXISTS alter_00121;
+DROP TABLE IF EXISTS alter_00121 SYNC;
```

Comment:
It does not really make sense to edit all tests and add `SYNC` modifier to every `DROP` query, you can simply enable `database_atomic_wait_for_drop_and_detach_synchronously` in you config (as it's enabled in CI for convenience)

Reply:
Thanks for mentioning it. I modified only tests I run locally. Setting `database_atomic_wait_for_drop_and_detach_synchronously` is a good approach, but in general, it's nice when a stuff just works.